bundles/org.openhab.binding.gardena/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -645,6 +711,10 @@
 		<item-type>String</item-type>
 		<label>RF Link State</label>
 		<description>The state of the RF link</description>
+		<tags>
+			<tag>Status</tag>
+			<tag>SignalStrength</tag>
```

Comment:
As this channel is unknown/online/offline, `SignalStrength` does not seem to be a good match. 

/not related to this PR, i wonder what the purpose is of this channel, as if the RF link would be offline, i guess the Thing would also be offline, no? This channels seems to sort of duplicate.

Reply:
> i guess the Thing would also be offline, no?
> This channels seems to sort of duplicate.

Your guess is as good as mine. I don't recall ever having seen Thing = Online combined with RF Line = Offline -- or vice versa, but I assume the original author had done that for a reason.